sw/host/opentitanlib/src/test_utils/lc_transition.rs

```diff
@@ -91,17 +92,16 @@ fn setup_lc_transition(
     }
 
     // If the transition requires a token, write it to the multi-register.
-    if let Some(token_words) = token {
-        let token_regs = [
-            &LcCtrlReg::TransitionToken0,
-            &LcCtrlReg::TransitionToken1,
-            &LcCtrlReg::TransitionToken2,
-            &LcCtrlReg::TransitionToken3,
-        ];
-
-        for (reg, value) in iter::zip(token_regs, token_words) {
-            jtag.write_lc_ctrl_reg(reg, value)?;
-        }
+    let token_words = token.expect("Get token").into_register_values();
```

Comment:
This is causing CI to fail. Why expect here?

Reply:
I did this due to a compiler suggestion.

But I ended up reverting the `trigger_lc_transition` interface change in `lc_transition` since it affects many other existing provisioning programs and I would like to avoid changing these at this point.